src/test/burn/WixToolsetTest.BurnE2E/CacheTests.cs

```diff
@@ -17,23 +18,35 @@ public class CacheTests : BurnE2ETests
         [Fact]
         public void CanCache5GBFile()
         {
-            var packageA = this.CreatePackageInstaller("PackageA");
-            var bundleC = this.CreateBundleInstaller("BundleC");
-
-            packageA.VerifyInstalled(false);
-
             // Recreate the 5GB payload to avoid having to copy it to the VM to run the tests.
+            const long FiveGB = 5_368_709_120;
+            const long OneGB = 1_073_741_824;
             var targetFilePath = Path.Combine(this.TestContext.TestDataFolder, "fivegb.file");
+
+            // If the drive has less than 5GB (for the test file) plus 1GB (for working space), then
+            // skip the test.
+            var drive = new DriveInfo(targetFilePath.Substring(0, 1));
+            if (drive.AvailableFreeSpace < FiveGB + OneGB)
+            {
+                Console.WriteLine("Skipping CanCache5GBFile() test because there is not enough disk space available to run the test.");
+                return;
```

Comment:
Ideally we would get the test marked as skipped instead of passing.

On a related note, the move to the mono repo lost the uploading of all the bundle logs as an artifact. Once we start running these tests in CI, it will be nearly impossible to diagnose failures.

Reply:
I wanted to mark it skipped but didn't know of a way to do that in XUnit while the test was running (admittedly I didn't look very hard last night).

Didn't mean to lose the bundle log upload. Can you provide more information (preferably in an issue so I don't lose it) and I'll fix it up? Diagnosing CI issues is challenging so I definitely want to make it easier.